package.json

```diff
@@ -55,11 +55,11 @@
         "typescript": "^3.8.3"
     },
     "dependencies": {
-        "@azure-tools/autorest-extension-base": "^3.1.235",
-        "@azure-tools/codegen": "^2.5.276",
+        "@azure-tools/autorest-extension-base": "3.1.235",
```

Comment:
Though we use the fixed version here, we still need to do the following in the future:
1. Regularly update the dependency libraries to catch up with their latest update/bug fixing
2. Understand the root cause on how that version upgrade causing the break.  We could send feedback to data plane team to avoid that.

Reply:
Agreed. But I found two of them has been renamed.
"@azure-tools/autorest-extension-base" => "@autorest/extension-base"
"@azure-tools/codemodel" => "@autorest/codemodel"
and we are not aware any of this changes.  It would be nice if they can notice us such kind of change.